src/main/java/me/partlysanestudios/partlysaneskies/config/oneconfig/OneConfigScreen.java

```diff
@@ -75,6 +75,19 @@ public void resetBrokenStrings() {
     )
     public boolean ignored;
 
+    @Dropdown(
+            name = "Update Channel",
+            subcategory = "Updates",
+            category = "General",
+            options = {
+                    "Release",
+                    "Pre-release",
+            }
+    )
+    public int releaseChannel = Integer.parseInt("@RELEASE_CHANNEL@");
```

Comment:
You sure you want it like that?

Reply:
What about that, it's something we change in Gradle.properties. That way if someone downloads the prerelease version, it automatically set it to the prerelease version channel

Reply:
Okay🤷